README.md

```diff
@@ -1,66 +1,92 @@
+[![License](https://img.shields.io/pypi/l/napari-stress.svg?color=green)](https://github.com/biapol/napari-stress/raw/master/LICENSE)
+[![PyPI](https://img.shields.io/pypi/v/napari-stress.svg?color=green)](https://pypi.org/project/napari-stress)
+[![Python Version](https://img.shields.io/pypi/pyversions/napari-stress.svg?color=green)](https://python.org)
+[![Anaconda-Server Badge](https://anaconda.org/conda-forge/napari-stress/badges/version.svg)](https://anaconda.org/conda-forge/napari-stress)

```

Comment:
This one doesn't display for me - https://anaconda.org/conda-forge/napari-stress requires a login?

Reply:
Fixed. I removed the button. We should bring it back once the plugin is on conda